packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -182,7 +182,11 @@ export default class VList {
      * @param shouldReuseAllAncestorListElements Optional - defaults to false.
      * @param disableListChain Whether we want to disable list chain functionality. @default false
      */
-    writeBack(shouldReuseAllAncestorListElements?: boolean, disableListChain?: boolean) {
+    writeBack(
+        shouldReuseAllAncestorListElements?: boolean,
+        disableListChain?: boolean,
+        newList?: boolean
```

Comment:
I'm always trying to avoid this kind of parameter. It is hard to say whether it is a new list from the caller side.

Can we just check if it is a new list within our code? For exmaple, if there is already a OL/UL and we reuse it, it should not be new list. Otherwise we can treat it as new?